in_toto/gpg/common.py

```diff
@@ -159,28 +165,56 @@ def parse_pubkey_bundle(data, keyid):
     the passed keyid.
 
   """
-  master_public_key = None
+  master_public_key = {}
   sub_public_keys = {}
 
   # Iterate over the passed public key data and parse out master and sub keys.
   # The individual keys' headers identify the key as master or sub key.
   packet_start = 0
+  expiration_datetime = 0
+  expires_after = None
+  #master_public_key = {}
```

Comment:
Why is this commented out?

Reply:
Sorry about this. I meant to remove that line, but forgot to. I'll do so in my next commit.